backends-common/elasticsearch/pom.xml

```diff
@@ -73,11 +73,6 @@
             <artifactId>slf4j-api</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.slf4j</groupId>
```

Comment:
So no logging for tests? Couldn't you include **logback-classic** with scope test?

Reply:
yes